tests/tools/test_text_to_classification.py

```diff
@@ -0,0 +1,12 @@
+from PIL import Image
+from vision_agent_tools.tools.text_to_classification import TextToClassification
+
+
+def test_successful_text_to_classification_tool():
+    test_image = "safework.jpg"
+    image = Image.open(f"tests/tools/data/nsfw_classification/{test_image}")
+
+    text_to_class = TextToClassification(model="nsfw_classification")
+    results = text_to_class(image)
```

Comment:
I run into this problem, not sure how we should handle while using the tool in another repo 
```python
self.detector = TextToObjectDetection(model=TextToObjectDetectionModel.OWLV2)
...
inferences = self.detector(prompts=prompts, image=image, confidence=confidence)
```

I get this error
```
TypeError: TextToObjectDetection.__call__() got an unexpected keyword argument 'confidence'
```
owlv2 has an extra parameter, not sure how we should handle this case? 
any idea @camiloaz @hugohonda

Reply:
let's ignore this, I noticed we are not sending custom fields to the tools, we'll use the default confidence